code/android-core-library/src/androidTest/java/com/adobe/marketing/mobile/internal/util/SQLiteDatabaseHelperTests.java

```diff
@@ -41,7 +41,9 @@ public class SQLiteDatabaseHelperTests {
 
     @Before
     public void setUp() {
-        dbPath = new File(InstrumentationRegistry.getInstrumentation().getContext().getCacheDir(), "test.sqlite").getPath();
+        File dbFile = ApplicationProvider.getApplicationContext().getDatabasePath("test.sqlite");
+        dbFile.delete();
```

Comment:
Can you move the deletion to dispose() funtion which gets called after each test